src/idiom80.fuzion

```diff
@@ -0,0 +1,2 @@
+x := -7.9
```

Comment:
remove this line here, the code should be just what is specified at [prgramming-idioms.org](https://www.programming-idioms.org/idiom/80/truncate-floating-point-number-to-integer)